app/views/recipes/public_recipes.html.erb

```diff
@@ -1,3 +1,15 @@
-<h1>Recipes#public_recipes</h1>
-<h1>list of publicrecipes</h1>
-<p>Find me in app/views/recipes/public_recipes.html.erb</p>
+<section class="recipe-list">
+  <h1>PUBLIC RECIPES</h1>
+  <% @public_recipes&.each do |recipe| %>
+    <div class="recipe-card">
+      <div>
+        <h3 calss="recipe-title"><%= link_to recipe.name, recipe_path(recipe.id), class:"recipe-title"%></h3>
```

Comment:
- [ ] [**Optional**] You have created a link for `recipe` detail well. My suggestion is we need to show guest user detail recipe information. because from requirement it says 
      1. If the recipe is public or the user is the owner of the recipe, should display the recipe details as in the wireframe.